includes/polling/os/transition.inc.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * LibreNMS
+ *
+ * Copyright (c) 2014 Neil Lathwood <https://github.com/laf/ http://www.lathwood.co.uk/fa>
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
+
+$objectID = preg_replace('/[\r\n\"]+/', ' ', snmp_get($device, ".1.0.8802.1.1.2.1.3.3.0", "-OQv"));
```

Comment:
\r \n and " are already trimmed in the snmp_get function, so no need to do it again.

Reply:
Fixed.